core/components/molecules/form/actions/actions.js

```diff
@@ -30,6 +30,7 @@ const Actions = props => {
           <ButtonGroup>
             {primaryAction && (
               <Button
+                {...primaryAction}
                 appearance="primary"
                 icon={primaryAction.icon}
                 onClick={primaryAction.handler}
```

Comment:
I know we recently changed this already, but if we're just going to proxy props, I wonder if it makes sense to just change `handler` to `onClick`?

Reply:
Makes sense